x-pack/test_serverless/api_integration/test_suites/observability/burn_rate_rule/burn_rate_rule.ts

```diff
@@ -42,7 +42,7 @@ export default function ({ getService }: FtrProviderContext) {
     let ruleId: string;
 
     before(async () => {
-      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+      roleAuthc = await svlUserManager.createM2mApiKeyWithRoleScope('admin');
```

Comment:
What does `M2m` stand for?

Reply:
Machine to machine. We wanted to be more specific about the intention - API keys are typically for machine to machine automations and apps, and cookies are typically for human interactive users.